sw/airborne/firmwares/rotorcraft/stabilization/stabilization_attitude_quat_indi.c

```diff
@@ -327,9 +336,11 @@ void lms_estimation(void)
   if (g_est.r < 0.01) { g_est.r = 0.01; }
   if (g2_est < 0.01) { g2_est = 0.01; }
 
-  //Commit the estimated G values and apply the scaling
-  g1.p = g_est.p * SCALE;
-  g1.q = g_est.q * SCALE;
-  g1.r = g_est.r * SCALE;
-  g2 = g2_est * SCALE;
+  if(use_adaptive_indi) {
```

Comment:
Don't want to nag about formatting all the time, but there should be a space between the if and the bracket...
I guess it doesn't hurt to run the ./fix_code_style.sh script on the file...